mods/d2k/metrics.yaml

```diff
@@ -7,3 +7,12 @@ Metrics:
 	FactionSuffix-smuggler: ordos
 	FactionSuffix-mercenary: ordos
 	TextfieldColorHighlight: 7f4d29
+	ChatLineSound: ChatLine
+	ClickDisabledSound: ClickDisabledSound
+	ClickSound: ClickSound
+	ChatMessageColor: FFFFFF
+	SystemMessageColor: FFFF00
+	NormalSelectionColor: FFFFFF
+	AltSelectionColor: 00FFFF
+	CtrlSelectionColor: FFFF00
```

Comment:
Those are all defined in the common metrics file, so shouldn't be needed here.

Reply:
That was a rebase resolved the wrong way. Sorry.